seqlike/SeqLike.py

```diff
@@ -176,7 +176,9 @@ def nt(self, auto_backtranslate=True, **kwargs) -> "SeqLike":
         """
         # Start with auto-back-translation
         if self._aa_record and self._nt_record is None and auto_backtranslate:
-            return self.back_translate(**kwargs)
+            nt_record = self.back_translate(**kwargs)
+            nt_record.annotations["molecule_type"] = "DNA"
```

Comment:
@ndousis it looks like the code changes here are intended to fix a bug - where the annotations are missing. If so, would you be kind enough to add the following test in an appropriate back-translate test?

```python
assert back_translated.annotations["molecule_type"] == "DNA"
```